### ISSUE TITLE
Urls for  packages  are now not  proper 

### ISSUE THREAD

**Issue by djkormo** [demotomohiro/remocolab#1]
File: https://svwh.dl.sourceforge.net/project/libjpeg-turbo/2.0.3/libjpeg-turbo-official_2.0.3_amd64.deb  can not be downloaded
File: https://svwh.dl.sourceforge.net/project/virtualgl/2.6.2/virtualgl_2.6.2_amd64.deb  can not be downloaded
File: https://svwh.dl.sourceforge.net/project/turbovnc/2.2.3/turbovnc_2.2.3_amd64.deb  can not be downloaded

Additionally please add try except in

def _download(url, path):
  try:
    with urllib.request.urlopen(url) as response:
      with open(path, 'wb') as outfile:
        shutil.copyfileobj(response, outfile)
  except:
    pass
    print('File:',url,' can not be downloaded')

**Comment by demotomohiro** [comment]
URLs in `remocolab.py` to download libjpeg-turbo, VirtualGL and TurboVNC are working.
I tested remocolab today and it worked.
But when I clicked URL in your comment, it doesn't start downloading the file.
Making new tab in my browser and put the URL start downloading the file.

If you want to download these files from your browser, following URL might be better.
https://sourceforge.net/projects/turbovnc/files/
https://sourceforge.net/projects/virtualgl/files/
https://sourceforge.net/projects/libjpeg-turbo/files/
